clients/vault/src/system.rs

```diff
@@ -679,15 +682,6 @@ impl VaultService {
 
 		tasks.append(&mut replace_tasks);
 
-		tasks.push((
```

Comment:
:scream: removing this duplicate, as it was original found in https://github.com/pendulum-chain/spacewalk/blob/e7a672ba1e21c98a70df30a6ee458317951dd597/clients/vault/src/system.rs#L623-L630
inside `fn create_initial_tasks(...`

----

Originally it runs after `Replace Cancellation Scheduler` and  before `Redeem Request Listener`:
https://github.com/pendulum-chain/spacewalk/blob/31df1832483199b978ae07f6ed4e018a11ff4dba/clients/vault/src/system.rs#L619-L639